packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -6,13 +6,17 @@ import type { BaseSelectOption } from "../types";
 import { areObjectsFlatEqual } from "../utils/objects";
 import { getFirstInvalidType, transformValidityStateToObject } from "../utils/validity";
 
-export type CustomErrorType = string | FormErrorMessages;
+export type CustomMessageType = string | FormMessages;
 
 export type CustomValidityProp = {
   /**
    * Custom error message to show. Will only show up after the user has interacted with the input.
    */
-  customError?: CustomErrorType;
+  customError?: CustomMessageType;
+  /**
+   * Custom success message to show. Will only show up after the user has interacted with the input.
+   */
+  customSuccess?: CustomMessageType;
```

Comment:
For errors there are native HTML error messages so we call this property `customError`.
For success tho, there are no default success messages, so I'd prefer to e.g. call this `successMessage`.

We could also think about refactoring our existing `message` and `messageTooltip` props to merge them to use a single `CustomMessageType` property